### ISSUE TITLE
PS4 version FFNT texture and animation load fail 

### ISSUE THREAD

**Issue by pgw00k** [Joschuka/fmt_g1m#8]
Hello,Joschka thank you for this tool!

1. i use this tool to load ps4 version FFNT file,if i turn on the g1t,it will load fail,but it could load mesh when i turn off load g1t.
2. FFNT(PS4) game looks like does not contain G1A or G2A animation file,i guess gapk and gmpk is the animation file,of course i notice gapk and gmpk may be filegroup?i need to split them to load or orther process？

anyways thanks your work！

```
Detected file type: Koei Tecmo KTGL Texture
Extra Header found: Size 12, Version 16
Loaded Texture 1 of 68; 2048x2048; Format 60; Size 2AB000; System B; Mips 10
Traceback (most recent call last):
  File "E:\Program Files\noesisv433\plugins\python\fmt_g1m.py", line 1510, in LoadRGBA
    processG1T(g1tBs)
  File "E:\Program Files\noesisv433\plugins\python\fmt_g1m.py", line 942, in processG1T
    textureData = rapi.imageDecodeDXT(textureData, width, height, format,0.0,1)
TypeError: imageDecodeDXT() takes at most 5 arguments (6 given)
Detected file type: Koei Tecmo KTGL Model
Extra Header found: Size 12, Version 16
Loaded Texture 1 of 68; 2048x2048; Format 60; Size 2AB000; System B; Mips 10
Traceback (most recent call last):
  File "E:\Program Files\noesisv433\plugins\python\fmt_g1m.py", line 1608, in LoadModel
    processG1T(g1tDataBs)	
  File "E:\Program Files\noesisv433\plugins\python\fmt_g1m.py", line 942, in processG1T
    textureData = rapi.imageDecodeDXT(textureData, width, height, format,0.0,1)
TypeError: imageDecodeDXT() takes at most 5 arguments (6 given)
```

**Comment by Joschuka** [comment]
-I made a mistake with the last update when I added support for another format, should hopefully be fixed with 48cb6ac . Let me know if this your g1t files still don't work. Also you may want to update Noesis since the imageDecodeDXT has been changed recently.

-Yes, gmpk are the models' containers and gapk are the animations' containers. If you want to extract them you can use my partner's tools : https://forum.xentax.com/viewtopic.php?f=10&t=21679 Just drag and drop the containers on the Cethleann.Unbundler.exe and it should extract the files.

**Comment by pgw00k** [comment]
Thanks!I change `bNormalized = True` load texture is OK.
and i use winhex to split a file to load animation is fine,nice tool!
